src/style.css

```
@@ -1 +1,8 @@
 /* styles go here */
+.field .form-field:not(:last-child) {
+  margin-bottom: 10px;
+}
```

Try to keep your selectors as simple as possible
```suggestion
.form-field:not(:last-child) {
  margin-bottom: 10px;
}
```